app/models/spree/roles_permission_set.rb

```diff
@@ -0,0 +1,6 @@
+module Spree
+  class RolesPermissionSet < ActiveRecord::Base
+    belongs_to :role
```

Comment:
@nimish-mehta Lets add missing indexes in all tables we have created or changed.